aea/configurations/data_types.py

```diff
@@ -309,17 +322,7 @@ def is_valid_str(cls, public_id_string: str) -> bool:
 
     @classmethod
     def from_str(cls, public_id_string: str) -> "PublicId":
-        """
-        Initialize the public id from the string.
-
-        >>> str(PublicId.from_str("author/package_name:0.1.0"))
-        'author/package_name:0.1.0'
-
-        A bad formatted input raises value error:
-        >>> PublicId.from_str("bad/formatted:input")
-        Traceback (most recent call last):
-        ...
-        ValueError: Input 'bad/formatted:input' is not well formatted.
```

Comment:
These are useful, why removed?

Reply:
This was because of some test which parsing the doctsrings and executing them, I thought I'll fix this when I update the tests: https://github.com/valory-xyz/open-aea/pull/103#discussion_r844248795

Reply:
Sounds good!